docs/docs/configuration/live.md

```diff
@@ -21,7 +21,14 @@ webRTC works by creating a websocket connection on extra ports. One of the follo
 * Frigate is run with `network_mode: host` to support automatic UDP port pass through locally and remotely. See https://github.com/AlexxIT/go2rtc#module-webrtc for more details
 * Frigate is run with `network_mode: bridge` and has:
     * Router setup to forward port `8555` to port `8555` on the Frigate device.
-    * For local webRTC, you will need to create your own go2rtc config:
+    * Added port to your docker ports list, for example:
```

Comment:
I think we need to elaborate on this a bit more. Explain that docker users need to map the ports and on the addon side these ports are disabled by default and need to be enabled in the addon config.

Reply:
actually I think we are going to pass 8555 by default since it is not exposing data when not being used